src/components/Layout/HomeContent.js

```diff
@@ -372,21 +376,22 @@ export function HomeContent() {
                       </div>
                       <div className="flex flex-col items-start justify-center pt-0 gap-3 px-2.5 lg:pt-8 lg:px-8">
                         <h4 className="leading-tight text-primary dark:text-primary-dark font-semibold text-3xl lg:text-4xl">
-                          Go truly native
+                          Torne-se verdadeiramente nativo
                         </h4>
                         <p className="h-full lg:text-xl text-secondary dark:text-secondary-dark leading-normal">
-                          People expect native apps to look and feel like their
-                          platform.{' '}
+                          As pessoas esperam que os aplicativos nativos se
+                          pareçam com sua plataforma.{' '}
                           <Link href="https://reactnative.dev">
                             React Native
                           </Link>{' '}
-                          and{' '}
+                          e{' '}
                           <Link href="https://github.com/expo/expo">Expo</Link>{' '}
-                          let you build apps in React for Android, iOS, and
-                          more. They look and feel native because their UIs{' '}
-                          <i>are</i> truly native. It’s not a web view—your
-                          React components render real Android and iOS views
-                          provided by the platform.
+                          permitem que você crie aplicativos em React para
```

Comment:
Acredito que aqui seria `permite` porque está se referenciando ao `Expo` ao invés de `As pessoas`.
```suggestion
                          permite que você crie aplicativos em React para
```